src/Config.cpp

```diff
@@ -230,11 +230,7 @@ std::vector<ConfigItem<int>> CFG::getIntItems()
         {"motion.frame_width", motion.frame_width, IVS_AUTO_VALUE, validateIntGe0},
         {"motion.frame_height", motion.frame_height, IVS_AUTO_VALUE, validateIntGe0},
         {"motion.monitor_stream", motion.monitor_stream, 1, validateInt1},
-        {"motion.roi_0_x", motion.roi_0_x, 0, validateIntGe0},
-        {"motion.roi_0_y", motion.roi_0_y, 0, validateIntGe0},
-        {"motion.roi_1_x", motion.roi_1_x, IVS_AUTO_VALUE, validateIntGe0},
-        {"motion.roi_1_y", motion.roi_1_y, IVS_AUTO_VALUE, validateIntGe0},
-        {"motion.roi_count", motion.roi_count, 1, [](const int &v) { return v >= 1 && v <= 52; }},
+        {"motion.roi_count", motion.roi_count, IVS_AUTO_VALUE, [](const int &v) { return v >= 1 && v <= 52; }},
```

Comment:
MM.. In any case, we must consider the variable in this validation.

`{"motion.roi_count", motion.roi_count, IVS_AUTO_VALUE, [](const int &v) { return v >= 1 && v <= IMP_IVS_MOVE_MAX_ROI_CNT; }},`

But I didn't add it because I had to add a device-dependent header just for this variable. I guess I should do it. 🤔